### ISSUE TITLE
workspace kontains no namespaces

### ISSUE THREAD

**Issue by theguywhoburns** [ratquaza/recipe-creator#1]

**Comment by theguywhoburns** [comment]
![Снимок экрана 2022-11-15 144534](https://user-images.githubusercontent.com/74488772/201911827-c097b041-ba29-40ba-bc30-7d15a830c1ef.png)

**Comment by ratquaza** [comment]
Hi Demiyanit, it seems you've opened the parent folder of the datapack, and not the datapack folder itself.
I've attached an example workspace that I'm in and the extension functions for me.
![image](https://user-images.githubusercontent.com/32216729/201931735-c7f47180-3c3c-419f-ad00-ef4bf965068e.png)

Instead of opening the parent folder, try selecting the folder I've highlighted with "Open Folder" in VS Code and see if the extension works.
![image](https://user-images.githubusercontent.com/32216729/201931980-3f4e6a51-5207-4a41-8626-dda375fc388a.png)
